src/bleak_esphome/backend/client.py

```diff
@@ -57,6 +54,21 @@
 CCCD_INDICATE_BYTES = b"\x02\x00"
 
 DEFAULT_MAX_WRITE_WITHOUT_RESPONSE = DEFAULT_MTU - GATT_HEADER_SIZE
+
+# BLE characteristic property bit masks
+PROPERTY_MASKS = {
```

Comment:
This can be replaced with `from bleak.assigned_numbers import CHARACTERISTIC_PROPERTIES`